pkg/drivemgr/basemgr/basemgr.go

```diff
@@ -56,19 +56,19 @@ func (mgr LinuxUtilManager) GetDrivesList() ([]*api.Drive, error) {
 	return devices, nil
 }
 
-//NewLinuxUtilManager is a constructor LinuxUtilManager
-func NewLinuxUtilManager(exec command.CmdExecutor, logger *logrus.Logger) *LinuxUtilManager {
-	return &LinuxUtilManager{
+//NewBaseManager is a constructor BaseManager
+func NewBaseManager(exec command.CmdExecutor, logger *logrus.Logger) *BaseManager {
```

Comment:
Maybe we should use "New" instead of "NewBaseManager" since it's the only one constructor in the package.
`basemgr.New(....)` looks short and concise.